test/view_component/integration_test.rb

```diff
@@ -320,18 +320,22 @@ def test_preview_from_multiple_preview_paths
     assert_select("div", "hello,world!")
   end
 
-  def test_renders_preview_source
+  def test_renders_preview_source_without_template
     get "/rails/view_components/preview_component/default"
 
     assert_select ".view-component-source-example h2", "Source:"
     assert_select ".view-component-source-example pre.source code"
+    refute_match '&lt;%=', response.body
+    refute_match '%&gt', response.body
```

Comment:
Wondering if there is a better way to ensure that the source is or isn't template code?